_posts/2019-05-06-browsertools#1-resource-timing-part1.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Browser Tools #1: Resource Timing (part 1)"
+categories: browsertools
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+read_time : 10
+excerpt: ""
+---
+
+This post is the start of a series of blog posts about browser tools. Among those we plan to look into any kind of tool in and around the browser, so mostly things web developers eventually find useful. Let's dive right in.
+
+Up to here this **page loaded <span id="num-assets-loaded-1">??</span> assets** (or resources) and **took <span id="time-taken-loading-1">??</span> seconds to load**. All those information were gathered, just now, via the Resource Timing interface, which we would like to cover in this article. <span id="loading-failed-hint-1">(If you just see "??" then reading the data didn't work, do you have an older browser? Anyways read on so you can try if and how the described API works in your browser?)</span> At the end of the article the same stats with the updated values can be found, watch out.
```

Comment:
How about ticking the name of the API used: `Resource Timing interface` - this will stand out nicely.

<img width="379" alt="Screenshot 2019-05-23 at 16 39 17" src="https://user-images.githubusercontent.com/1819138/58261840-5d385b00-7d79-11e9-9cb7-1b90e515e1e8.png">